opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -590,6 +590,113 @@ mod tests {
         );
     }
 
+    // "multi_thread" tokio flavor must be used else flush won't
+    // be able to make progress!
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    #[ignore = "Bug bug."]
```

Comment:
don't have a tracking issue. There are open PRs touching aggregations, and more to come, and will review how best to fix this considering upcoming/pending changes.

Reply:
Can we add a tracking issue if not already there? And add more details why is this being ignored. I assume it is related to discussion here - https://cloud-native.slack.com/archives/C03GDP0H023/p1709591127115979

Reply:
There should be a way to ignore the specific assertion, rather than whole test, probably we can figure that separately.

Reply:
I think it's good to have tracking issues for things not working, even if we plan to revisit the design.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/1598